app/models/step_type.rb

```diff
@@ -76,31 +74,29 @@ def all_runners_files
   end
 
   def all_step_actions
-    [#all_background_steps_files,
+    [ # all_background_steps_files,
       all_inferences_files,
-      all_runners_files].flatten
+      all_runners_files
+    ].flatten
   end
 
   def task_type
-    return 'background_step' if (actions.count > 0)  || (step_action.nil?)
-    return 'cwm' if step_action && step_action.end_with?('.n3')
+    return 'background_step' if (actions.count > 0) || step_action.nil?
+    return 'cwm' if step_action&.end_with?('.n3')
+
     return 'runner'
   end
 
   def class_for_task_type
-    if task_type=='cwm'
+    if task_type == 'cwm'
       Steps::BackgroundTasks::Inference
-    elsif task_type=='runner'
+    elsif task_type == 'runner'
       Steps::BackgroundTasks::Runner
-    elsif task_type=='background_step'
-      Step
-      #["BackgroundSteps::", step_action.gsub(".rb","").classify].join.constantize
```

Comment:
I removed this in my initial clean-up of the code when I realised it wasn't used. However with hindsight it would probably have made more sense to just start using it.